cmd/config/options.go

```diff
@@ -0,0 +1,11 @@
+package config
```

Comment:
Should this be a little bit further down inside of `gitops` too? 🤔

Reply:
There is a `cmd/internal` directory for some "CLI library code", this might go well in there.

Reply:
@foot @ozamosi I have moved the config package to `cmd/internal`. @ozamosi If you are happy to approve can you please also merge, thanks!